src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/Controllers/DataSetsController.cs

```diff
@@ -38,6 +38,25 @@ public DataSetsController(IDataSetService dataSetService)
             .HandleFailuresOrOk();
     }
 
+    /// <summary>
+    /// Get a data set version
+    /// </summary>
+    /// <remarks>
+    /// Get a data set version, including a full list of its changes.
```

Comment:
Looks like this description is wrong. We don't get the full list of changes here anymore as we'll have a separate endpoint for this.

We could adjust this to:

```
Get a data set version's summary details.
```